src/MinFraud.php

```diff
@@ -36,17 +34,10 @@
  *
  * If the request fails, an exception is thrown.
  */
-class MinFraud
+class MinFraud extends ServiceClient
 {
-    const VERSION = 'v1.12.0';
```

Comment:
We should update `dev-bin/release.sh` for this change (and presumably that `sed` fix as well).